src/fbRemoteConfig.ts

```diff
@@ -0,0 +1,66 @@
+import { asBoolean, asObject, asOptional, asValue, uncleaner } from 'cleaners'
+import { makeReactNativeDisklet } from 'disklet'
+
+import { REMOTE_CONFIG_STICKY } from './constants/constantSettings'
+
+export const asFbRemoteConfig = asObject({
+  swipeLastUsp: asOptional(asBoolean, true),
+  createAccountType: asOptional(asValue('full', 'light'), 'full')
+})
+
+export type FbRemoteConfig = ReturnType<typeof asFbRemoteConfig>
+
+const stickyRemoteConfigDisklet = makeReactNativeDisklet()
+
+const stickyDistribution = {
+  swipeLastUsp: 0.5,
+  createAccountType: 0.1
+}
+
+let stickyRemoteConfig: FbRemoteConfig | null = null
+
+/**
+ * Initializes the local 'sticky remote config' file containing the randomly
+ * generated variant values. This is used for variant values that are required
+ * prior to the initialization of the fetched remote config.
+ * Once generated, values 'stick' until the remote config type changes.
+ */
+export const initializeStickyRemoteConfig = async () => {
+  try {
+    const stickyRemoteConfigJson = await stickyRemoteConfigDisklet.getText(REMOTE_CONFIG_STICKY)
+    stickyRemoteConfig = asFbRemoteConfig(JSON.parse(stickyRemoteConfigJson))
+  } catch (err) {
+    // Not found or mismatched. Re-generate with random values according to the
+    // defined distribution.
+    const generatedRemoteConfigSticky: FbRemoteConfig = {
+      swipeLastUsp: Math.random() < stickyDistribution.swipeLastUsp,
+      createAccountType: Math.random() < stickyDistribution.createAccountType ? 'light' : 'full'
+    }
+    const generatedJsonData = JSON.stringify(uncleaner(asFbRemoteConfig)(generatedRemoteConfigSticky))
```

Comment:
uncleaner not necessary. The type is derived from the cleaner so that ensures type compatibility